docs/i18n.rst

```diff
@@ -225,7 +225,12 @@ First, you'll need to have GNU ``gettext`` available on your path. You may be ab
 
 .. note:: If you install ``gettext`` on Mac with Homebrew, you may need to add the binary to your path manually
 
-Next, ensure you have an environment variable ``CROWDIN_API_KEY`` set to the Learning Equality organization `account API key <https://support.crowdin.com/api/api-integration-setup/>`__.
+Next, ensure you have the following environment variables set:
+
+* ``CROWDIN_API_KEY`` - your `account API key <https://support.crowdin.com/api/api-integration-setup/>`.
+* ``CROWDIN_LOGIN`` - your `Crowdin username <https://crowdin.com/settings#account>`.
+* ``CROWDIN_PROJECT`` - (optional, default: "kolibri") the "Project Identifier" on Crowdin that you want to work with. You can find this on Crowdin in the project's settings in the "API & Webhooks" section. 
+* ``CROWDIN_LOCALE_ABSOLUTE_PATH`` - (optional) set where you want the generated files to appear (eg "kolibri/locale" or "contentcuration/locale").
```

Comment:
optional, when?

Reply:
Will improve with clarifying error messages. Not optional.